stats.php

```diff
@@ -15,7 +15,14 @@
 
 // Limit tracking to production
 if ( true === WPCOM_IS_VIP_ENV && false === WPCOM_SANDBOXED ) {
-	add_action( 'transition_post_status', __NAMESPACE__ . '\track_publish_post', 9999, 2 );
+	
+	// Record created time of a post
+	add_action( 'save_post', __NAMESPACE__ . '\record_created_time', 9999, 3 );
+	
+	// Record published time and send telemetry when post is published
+	add_action( 'transition_post_status', __NAMESPACE__ . '\record_published_time', 9999, 2 );
```

Comment:
```suggestion
	add_action( 'transition_post_status', __NAMESPACE__ . '\record_published_time', 9999, 3 );
```
since it's taking 3 params